README.md

```diff
@@ -10,7 +10,7 @@ See: https://github.com/fabric8io/fabric8-test/blob/master/ee_tests/README.md
 
 ## API tests on OSiO
 
-Our API tests use [pyresttest](https://github.com/svanoort/pyresttest.git) a pyton test framework that make all your API test declarative. No need to learn python, your test are plain old yaml file!
+Our API tests use [pyresttest](https://github.com/svanoort/pyresttest.git) a Python test framework that make all your API test declarative. No need to learn python, your test are plain old yaml file!
```

Comment:
Python test framework that make**s** 
all your API test**s**
no need to learn **P**ython
your test**s**

Reply:
nice, TY